test/mod/test_loop.py

```diff
@@ -0,0 +1,63 @@
+#
+# Test for the loop.py
+#
+
+import os
+
+
+from tempfile import TemporaryDirectory
+
+import pytest
+
+from osbuild import loop
+
+from ..test import TestBase
+
+
+@pytest.fixture(name="tempdir")
+def tempdir_fixture():
+    with TemporaryDirectory(prefix="lvm2-") as tmp:
```

Comment:
typo? prefix should probably be something like 'looptest-' so it's obvious what created it.